src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/HearingChannel.java

```diff
@@ -7,10 +7,11 @@
 @RequiredArgsConstructor
 public enum HearingChannel {
 
-    TELEPHONE("telephone", "Telephone", "Ffôn"),
-    VIDEO("video", "Video", "Fideo"),
-    FACE_TO_FACE("faceToFace", "Face To Face", "Wyneb yn wyneb"),
-    NOT_ATTENDING("notAttending", "Not Attending", null);
+    TEL("telephone", "Telephone", "Ffôn"),
```

Comment:
Is the key not supposed to be updated no the enum name?

Reply:
Corrected